composer.json

```diff
@@ -17,15 +17,18 @@
   "require": {
     "php": "^5.6.0 || ^7.0.0",
     "rokka/client": "^1.0.0",
-    "symfony/console" : "^2.7.0 || ^3.0.0",
-    "symfony/finder" : "^2.7.0 || ^3.0.0",
-    "symfony/config": "^2.7.0 || ^3.0.0",
-    "symfony/dependency-injection": "^2.7.0 || ^3.0.0",
-    "symfony/yaml": "^2.7.0 || ^3.0.0"
+    "symfony/console" : "^2.7.0 || ^3.0.0 || ^4.0.0",
+    "symfony/finder" : "^2.7.0 || ^3.0.0 || ^4.0.0",
+    "symfony/config": "^2.7.0 || ^3.0.0 || ^4.0.0",
+    "symfony/dependency-injection": "^2.7.0 || ^3.0.0 || ^4.0.0",
+    "symfony/yaml": "^2.7.0 || ^3.0.0 || ^4.0.0"
   },
   "require-dev": {
     "friendsofphp/php-cs-fixer": "^2.1"
   },
+  "conflicts": {
+    "rokka/client-bundle": "<=1.0.1"
```

Comment:
if we would run this lib with an old version of the bundle in symfony 3.4/4, the command attribute on the console.command tag would lead to errors with the console setup.